CoreWidgetProvider/Widgets/Templates/SSHWalletPatternMatching.json

```diff
@@ -0,0 +1,57 @@
+{
+  "type": "AdaptiveCard",
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "version": "1.5",
+  "body": [
+    {
+      "type": "Container",
+      "items": [
+        {
+          "type": "TextBlock",
+          "text": "Your SSH key contains pattern matching. Fill out the pattern to connect.",
```

Comment:
This should be localized

Reply:
Good catch. Will add to resw and test.

Reply:
How can I properly test if my reference to resw is working? I have added the string here: 

<img width="872" alt="image" src="https://github.com/microsoft/devhome/assets/98557455/7d61cbff-9811-4a14-83d5-e62db78ef215">

But when I debug the app it doesn't seem to be finding the correct string. Any tips for how to properly ensure I reference resw?

Reply:
I did not notice the Resources.cs file - I have fixed it.